### PR TITLE
feat(RulesTable): RHICOMPL-2083 - Update Ansible column to "Remediate"

### PR DESCRIPTION
**Pull request by bastilian**
This changes the "Ansible" column to be named "Remediation" and more user friendly column values.

Before:

Screenshot 2021-11-11 at 14 04 47

After: 

Screenshot 2021-11-11 at 13 51 18

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by skateman** [comment]
@bastilian can you please also put a before screenshot?

File: src/PresentationalComponents/RulesTable/Cells.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ export const Passed = ({ compliant }) =>
   );
 Passed.propTypes = ruleProps;
 
-export const Ansible = ({ remediationAvailable }) =>
-  remediationAvailable ? <CheckCircleIcon className="ins-u-passed" /> : 'No';
-Ansible.propTypes = ruleProps;
+export const RemediatonColumnCell = ({ remediationAvailable }) => (
+  <RemediationCell hasPlaybook={remediationAvailable} />
```
Comment (vkrizan) [inline src/PresentationalComponents/RulesTable/Cells.js:80]: Is the wrapper necessary?

**Review by vkrizan — COMMENTED** [review]
The end result looks good.

This PR and it's one commit does at least three things:
* changes the button and column text and visuals
* fixes the column render/responsiveness
* refactors the table to use default columns (instead of declaring them inline)

Each of those might be separated, as they are different concerns.

File: src/PresentationalComponents/RulesTable/Cells.js (hunk 1 above)
Comment (bastilian) [reply src/PresentationalComponents/RulesTable/Cells.js:80]: Yes.

**Comment by bastilian** [comment]
> Each of those might be separated, as they are different concerns.

They could be, yes, but the PR is so small and it would get squashed when merging to master anyways.

**Review by marleystipich2 — APPROVED** [review]
These updates look great and it looks like they fulfill all the requirements on the ticket. I'm not sure if you were still considering splitting it into a few different commits but I figured I would approve anyway. Thanks, @bastilian!

**Pull request merged**